src/wovnio/html/HtmlConverter.php

```diff
@@ -84,6 +100,37 @@ private function insertSnippet($dom) {
     }
   }
 
+  /**
+   * Insert hreflang tags for all supported_langs
+   *
+   * @param simple_html_dom_node $dom
+   */
+  public function insertHreflangTags($dom) {
+    $lang_codes = $this->store->settings['supported_langs'];
+    foreach ($dom->find('link') as $node) {
+      $hreflangValue = $node->getAttribute('hreflang');
+      if (in_array(Lang::getCode($hreflangValue), $lang_codes)) {
```

Comment:
I think here `$lang_code` should be iso normalized too

Reply:
I modified Lang::getCode to support Normalized values